### ISSUE TITLE
Node-RED fails on Ubuntu Server 18.04: node-red.service.j2 needs /usr/bin/node-red-pi NOT /usr/local/bin

### ISSUE THREAD

**Issue by holta** [iiab/iiab#1431]
@m-anish can you please help investigate?  Node-RED's service is failing a fresh install of IIAB 6.7/master (with PR #1430 for Node 10.x & the new Sugarizer) on a fresh Ubuntu Server 18.04 VM:
```
root@box:~# systemctl restart node-red
root@box:~# systemctl status node-red
● node-red.service - Node-RED
   Loaded: loaded (/etc/systemd/system/node-red.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Tue 2019-01-29 16:12:34 EST; 374ms ago
  Process: 2379 ExecStart=/usr/local/bin/node-red-pi --max-old-space-size=128 -v (code=exited, status=203/EXEC)
 Main PID: 2379 (code=exited, status=203/EXEC)

Jan 29 16:12:34 box.lan systemd[1]: node-red.service: Service hold-off time over, scheduling restart.
Jan 29 16:12:34 box.lan systemd[1]: node-red.service: Scheduled restart job, restart counter is at 5.
Jan 29 16:12:34 box.lan systemd[1]: Stopped Node-RED.
Jan 29 16:12:34 box.lan systemd[1]: node-red.service: Start request repeated too quickly.
Jan 29 16:12:34 box.lan systemd[1]: node-red.service: Failed with result 'exit-code'.
Jan 29 16:12:34 box.lan systemd[1]: Failed to start Node-RED.
```

**Comment by holta** [comment]
Conversely/Strangely: Node-RED (http://box:1880 and its systemd service node-red) both continue to work on RPi after I upgraded Node.js 8.x to 10.x (and rebooted).

I do not understand why this same upgrade (Node.js 8.x to 10.x with the new Sugarizer, at PR #1430) somehow caused Node-RED to fail on Ubuntu 18.04 :/

**Comment by holta** [comment]
On Ubuntu Server 18.04, `journalctl -u node-red` shows:
```
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Service hold-off time over, scheduling restart.
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Scheduled restart job, restart counter is at 4.
Jan 29 16:55:13 box.lan systemd[1]: Stopped Node-RED.
Jan 29 16:55:13 box.lan systemd[1]: Started Node-RED.
Jan 29 16:55:13 box.lan systemd[523]: node-red.service: Failed to execute command: No such file or directory
Jan 29 16:55:13 box.lan systemd[523]: node-red.service: Failed at step EXEC spawning /usr/local/bin/node-red-pi: No such file or directory
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Main process exited, code=exited, status=203/EXEC
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Failed with result 'exit-code'.
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Service hold-off time over, scheduling restart.
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Scheduled restart job, restart counter is at 5.
Jan 29 16:55:13 box.lan systemd[1]: Stopped Node-RED.
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Start request repeated too quickly.
Jan 29 16:55:13 box.lan systemd[1]: node-red.service: Failed with result 'exit-code'.
Jan 29 16:55:13 box.lan systemd[1]: Failed to start Node-RED.
```
Indeed http://box:1880 starts to work if /usr/local/bin/node-red-pi is changed to /usr/bin/node-red-pi  in /etc/systemd/system/node-red.service on Ubuntu Server 18.04.

Conclusion: the conditional https://github.com/iiab/iiab/blob/master/roles/nodered/templates/node-red.service.j2#L6 needs to be fixed: @m-anish are there any OS's that truly/still require /usr/local/bin here?

**Comment by holta** [comment]
Effectively this is a bug introduced by the earlier PR #1430, which the revised #1430 now fixes, by including PR #1432.
